### PR TITLE
adjusted nav and js

### PR DESCRIPTION
**Pull request by gozoslash**
@osmanzeki 

- I adjusted the html to fit the nav, mobile nav and backdrop as well as header in the body
- I adjusted the js to be able to close the dropdown when clicking on the document. I took inspiration from: https://stackoverflow.com/questions/14188654/detect-click-outside-element-vanilla-javascript
- I adjusted indentation in the css file because for some reason the clone was off 
- I also adjusted by 2px the way the caret is dispayed on the dropdown because ça me gossait en osti!

Can you please review :)

### CODE REVIEW

**Comment by osmanzeki** [comment]
You know what. Since you have `backdrop` class that occupies the whole screen when your menu is open, you could put your click event "outside" of the menu and "inside" the backdrop to trigger closing of the menu. Definitely try that.

File: shared.js [hunk 1]
```diff
@@ -31,9 +31,16 @@ backdrop.addEventListener('click', () => {
...
-    	dropdown.classList.add ('flex');
-    }
+  if (dropdown.classList.contains('flex')) {
+  	dropdown.classList.remove('flex');
+  } else {
+  	dropdown.classList.add ('flex');
+  }
+});
+
+document.addEventListener('click', (event) => {
```
Comment (osmanzeki) [inline shared.js]: So here you could probably target the `backdrop` element instead of the whole document.

Comment (gozoslash) [reply shared.js]: So I checked but in fact the backdrop is display none on desktop. Appears only on mobile but on mobile there is no dropdown so I have to target the document on desktop.

Comment (osmanzeki) [reply shared.js]: Ok makes sense to use what you had then.

**Pull request merged**